### DISCUSSION TITLE
无法支付且收不到邮箱登录验证码

### DISCUSSION THREAD

**Discussion by ShawnAlv** [getlantern/lantern-forum-cn#748] [Q&A]
支付界面提示输入邮箱以继续购买，输入后出现图标一直转圈加载，无论等多久一直这样。
同时邮箱登录也收不到验证码

**Comment by MaxMKMax** [comment]
@ShawnAlv 您好。我可以知道为什么您无法续订，因为您没有看到付款选项，或者是否存在阻止您付款的错误或付款限制? 您介意告诉我确切的问题并为我提供一些屏幕截图吗？太感谢了。

**Comment by MaxMKMax** [comment]
@ShawnAlv 您好. 应该是没购买成功所以没收到验证码.

**Comment by MaxMKMax** [comment]
@ShawnAlv 
您好. 我们的技术部门说您可以试看我们最新的测试版本. 应该可以帮助到您解决问题.
https://github.com/getlantern/lantern
